package.json

```
@@ -64,9 +64,9 @@
     "rimraf": "^3.0.0",
     "strip-ansi": "^7.0.0",
     "tape": "^5.0.0",
-    "tsd": "^0.20.0",
+    "tsd": "^0.22.0",
     "type-coverage": "^2.0.0",
-    "typescript": "^4.0.0",
+    "typescript": ">=4.7",
```

```suggestion
    "typescript": "^4.0.0",
```

Can you revert this?
The style we use here is to have relaxed ranges except when they are really needed.

---

I think it's truly needed in this case. It would allow TS 4.6 and older otherwise, which can't build this project.

---

It’s not needed. The TypeScript dependency isn’t different from any other. Version `^4.0.0` will resolve to TypeScript 4.7 (at the moment of writing). If you run into issues, try deleting `node_modules` and running `npm install`.